index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Technical Documentation Page</title>
+    <link rel = "stylesheet" href = "style.css">
+</head>
+<body>
+    <nav id = "navbar">
+        <header>JS DOCUMENTATION</header>
+        <ul>
+            <li><a class = "nav-link" href = "#Introduction">Introduction</a></li>
+            <li><a class = "nav-link" href = "#What_you_should_already_know">What you should already know</a></li>
+            <li><a class = "nav-link" href = "#Javascript_and_java">Javascript and java</a></li>
+            <li><a class = "nav-link" href = "#Hello_world">Hello world</a></li>
+            <li><a class = "nav-link" href = "#Variables">Variables</a></li>
+            <li><a class = "nav-link" href = "#Declaring_Variables">Declaring variables</a></li>
+            <li><a class = "nav-link" href = "#Variable_scope">Variable scope</a></li>
+            <li><a class = "nav-link" href = "#Global_variables">Global variables</a></li>
+            <li><a class = "nav-link" href = "#Constants">Constants</a></li>
+            <li><a class = "nav-link" href = "#Data_types">Data types</a></li>
+            <li><a class = "nav-link" href = "#If...else_statement">If...else statement</a></li>
+            <li><a class = "nav-link" href = "#While_statement">While statement</a></li>
+            <li><a class = "nav-link" href = "#Function_declarations">Function declarations</a></li>
+            <li><a class = "nav-link" href = "#Reference">Reference</a></li>
```

Comment:
Please make sure not to leave spaces when defining class for HTML elements. kindly correct them in your HTML tags where class or id is defined.